monkey/monkey_island/cc/environment/server_config_handler.py

```diff
@@ -20,20 +19,9 @@ def write_default_server_config_to_file(path: str) -> None:
     Path(path).write_text(default_config)
 
 
-def load_server_config_from_file(server_config_path):
+def load_server_config_from_file(server_config_path) -> IslandConfigOptions:
     with open(server_config_path, "r") as f:
         config_content = f.read()
         config = json.loads(config_content)
-        add_default_values_to_config(config)
 
-        return config
-
-
-def add_default_values_to_config(config):
-    config["data_dir"] = os.path.abspath(
-        os.path.expanduser(os.path.expandvars(config.get("data_dir", DEFAULT_DATA_DIR)))
-    )
-
-    config.setdefault("log_level", DEFAULT_LOG_LEVEL)
-
-    return config
```

Comment:
Are we deliberately removing this or is this part of the rebase mess?

Reply:
This commit also removes `monkey/tests/unit_tests/monkey_island/cc/environment/test_server_config_handler.py` as part of the rebase. Depending on this comment, we need to re-add/modify this file too.

Reply:
Deliberately. Config is now built in `island_config_options.py`.